### PR TITLE
SR-IOV: increase the verification timeout if SR-IOV is present

### PR DESCRIPTION
**Pull request by ffmancera**
Certain drivers like i40e take a long time to modify the VFs in the
kernel. Nmstate is timing out on verification because of that. In order
to fix this, nmstate is incresing the verification time if SR-IOV is
present on the desired state.

Signed-off-by: Fernando Fernandez Mancera <ffmancera@riseup.net>

### CODE REVIEW

**Review by cathay4t — APPROVED** [review]

File: libnmstate/netapplier.py [hunk 1]
```diff
@@ -109,7 +110,13 @@ def _apply_ifaces_state(plugins, net_state, verify_change, save_to_disk):
         plugin.apply_changes(net_state, save_to_disk)
     verified = False
     if verify_change:
-        for _ in range(VERIFY_RETRY_TIMEOUT):
+        if _net_state_contains_sriov_interface(net_state):
+            # If SR-IOV is present, the verification timeout is being increased
+            # to avoid timeouts due to slow drivers like i40e.
+            verify_retry = VERIFY_RETRY_TIMEOUT * 4
```
Comment (tyll) [inline libnmstate/netapplier.py]: `4` should be a constant. And why 4?

Comment (ffmancera) [reply libnmstate/netapplier.py]: We are retrying up to 4 times more. Yes, I will set it as a constant. 20 seconds should be enough time.

Comment (ffmancera) [reply libnmstate/netapplier.py]: Fixed!

**Comment by cathay4t** [comment]
oVirt has confirmed the fix. Merging.

**Comment by cathay4t** [comment]
The CI is failed on `AttributeError: 'NoneType' object has no attribute 'props'`

**Pull request merged**